src/java.net.http/share/classes/jdk/internal/net/http/ExchangeImpl.java

```diff
@@ -67,6 +69,14 @@ final Exchange<T> getExchange() {
         return exchange;
     }
 
+    final void setExpectTimeoutRaised(boolean timeoutRaised) {
```

Comment:
Nit - I don't think we will ever be calling this method with a value of `false`. So maybe we should just make this a no-arg method which in its implementation will set the flag to true? If you prefer it in the current form, that's fine too.

Reply:
I wouldn't be opposed to that, I can include it with the next commit